frontend/src/scenes/pipeline/hogfunctions/filters/HogFunctionFilters.tsx

```diff
@@ -110,41 +110,38 @@ export function HogFunctionFilters({ embedded = false }: { embedded?: boolean })
                                     mappings.
                                 </p>
                             )}
-                            {isTransformation && (
-                                <LemonBanner type="info">
-                                    For transformations, only event properties and SQL expressions can be used in
-                                    filters as they run during ingestion before other properties are available.
-                                </LemonBanner>
-                            )}
                             {!isTransformation && (
-                                <>
-                                    <TestAccountFilterSwitch
-                                        checked={filters?.filter_test_accounts ?? false}
-                                        onChange={(filter_test_accounts) =>
-                                            onChange({ ...filters, filter_test_accounts })
-                                        }
-                                        fullWidth
-                                    />
-                                    <PropertyFilters
-                                        propertyFilters={(filters?.properties ?? []) as AnyPropertyFilter[]}
-                                        taxonomicGroupTypes={[
-                                            TaxonomicFilterGroupType.EventProperties,
-                                            TaxonomicFilterGroupType.PersonProperties,
-                                            TaxonomicFilterGroupType.EventFeatureFlags,
-                                            TaxonomicFilterGroupType.Elements,
-                                            TaxonomicFilterGroupType.HogQLExpression,
-                                            ...groupsTaxonomicTypes,
-                                        ]}
-                                        onChange={(properties: AnyPropertyFilter[]) => {
-                                            onChange({
-                                                ...filters,
-                                                properties,
-                                            })
-                                        }}
-                                        pageKey={`HogFunctionPropertyFilters.${id}`}
-                                    />
-                                </>
+                                <TestAccountFilterSwitch
```

Comment:
I think above on the LemonField we should add an info when its not a mapping like

```
                info={useMapping ? 'Filters applied to all events before they reach a mapping' : 'Filters applied to all events'}

```